README.md

```diff
@@ -8,16 +8,397 @@ so don't try to use it for anything close to a production environment.
+
+```bash
+$ oc apply -f sample-oran2ims-metadata.yaml 
+orano2ims.oran.openshift.io/sample created
+```
+
+Then, a metadata-server Pod is running on the oran-o2ims namespace and a route created with the value included in the previous custom resource, 
+specifically in the spec.ingressHost field. That’s the URI where the IMS API will be listening from outside the OpenShift cluster:
+
+```bash
+$ oc get pods -n oran-o2ims
+NAME                              READY   STATUS    RESTARTS   AGE
+metadata-server-c655b559b-6chmx   1/1     Running   0          12s
+
+$ oc get route -n oran-o2ims
+NAME        HOST/PORT                                                        PATH                                                   SERVICES                    PORT   TERMINATION          WILDCARD
+api-27qrj   oran-o2ims.apps.hub0.inbound-int.se-lab.eng.rdu2.dc.redhat.com   /                                                      metadata-server             api    reencrypt/Redirect   None
+```
+
+Before querying the Metadata Server we need to allow that request. 
+Therefore we need to create a token associated with an already created serviceAccount named `client`:
+
+```bash
+$ export CLIENT_TOKEN=$(oc create token -n oran-o2ims client --duration=24h)
+$ export API_URI=$(oc -n oran-o2ims get route -o jsonpath={.items[0].spec.host})
+```
+
+##### Requests Examples
+
+###### GET API version
+
+To obtain the API version deployed:
+
+```bash
+$ curl --insecure --silent --header "Authorization: Bearer $CLIENT_TOKEN" 
+"https://${API_URI}/o2ims-infrastructureInventory/v1/api_versions" | jq
+```
+
+###### GET O-Cloud information
+
+The next query obtains O-Cloud information:
+
+```bash
+$ curl --insecure --silent --header "Authorization: Bearer $CLIENT_TOKEN" 
+"https://${API_URI}/o2ims-infrastructureInventory/v1" | jq
+```
+
+####  1.2.2. <a name='Deploymentmanagerserver'></a>Deployment manager server
+
+The deployment manager server needs to connect to kubernetes API of the
+RHACM hub. If you are already connected to an OpenShift cluster that has
+RHACM installed and configured you can obtain the token like this:
+
+```
+$ export BACKEND_TOKEN=$(oc create token -n open-cluster-management multicluster-operators --duration=24h)
+```
+Next, enable and configure the `deploymentManagerServerConfig` section of the ORANO2IMS CR yaml file.
+
+```yaml
+apiVersion: oran.openshift.io/v1alpha1
+kind: ORANO2IMS
+metadata:
+  annotations:
+  labels:
+    app.kubernetes.io/created-by: oran-o2ims
+    app.kubernetes.io/instance: orano2ims-sample
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: orano2ims
+    app.kubernetes.io/part-of: oran-o2ims
+  name: sample
+  namespace: oran-o2ims
+spec:
+  alarmSubscriptionServerConfig:
+    enabled: false
+  cloudId: c0332915-6bff-4d8d-8628-0ab3cc2c7e5e
+  deploymentManagerServerConfig:
+    backendToken: ${BACKEND_TOKEN}
+    backendType: regular-hub
+    enabled: true
+  image: quay.io/openshift-kni/oran-o2ims:4.16.0
```

Comment:
The `image` is also not necessary since the controller will inherit the value passed into its `IMAGE` environment variable which is part of the default deployment and bundled deployment.